<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassificatorLearningDialog</class>
 <widget class="QDialog" name="ClassificatorLearningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обучение классификатора</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Обучающая выборка</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_Files">
         <property name="title">
          <string>Список файлов</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QListWidget" name="listFiles"/>
          </item>
          <item>
           <widget class="QPushButton" name="buttonAddFile">
            <property name="text">
             <string>Добавить файл</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRemoveFile">
            <property name="text">
             <string>Убрать файл</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Classes">
         <property name="title">
          <string>Список классов</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QListWidget" name="listClasses"/>
          </item>
          <item>
           <widget class="QPushButton" name="buttonAddClass">
            <property name="text">
             <string>Добавить класс</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRemoveClass">
            <property name="text">
             <string>Удалить класс</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Разметка данных</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listLabels"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="buttonLoadLabelingInfo">
           <property name="text">
            <string>Загрузить разметочную информацию</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonChangeLabel">
           <property name="text">
            <string>Изменить выделенный</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSaveLabelingInfo">
           <property name="text">
            <string>Сохранить разметочную информацию</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="buttonTrainClassificator">
       <property name="text">
        <string>Тренировать классификатор</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="buttonSaveClassificatorDump">
       <property name="text">
        <string>Сохранить дамп классификатора в файл</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
